### ISSUE TITLE
validator:status command should clarify the meaning of signing blocks

### ISSUE THREAD

**Issue by nategraf** [celo-org/celo-monorepo#2234]
The validator status command mentioned signed blocks, which led a validator to be concerned that their validator is misconfigured, when in-fact there the signal reported it inconclusive to that respect. This should be clarified to avoid having to repost the following explanation.

> When is says "signed" in that context it specifically means that your signature was included in the ParentSeal which is aggregated during the the proposal of the next block. Essentially every block has a little over 2/3rds direct signatures, and as soon as the proposer has that quorum, the block is finalized. During the period when the next block is being proposed, signatures that arrive for the previous block after finalization are aggregated to for m the ParentSeal which is what is used for uptime. All that being said, there is no way that your signature will make it into the ParentSeal, so when calculating your uptime score each validator is only counted as being down if their signature does not appear in a number of consecutive blocks, by default 12.
TL;DR; The "signed blocks" indicator will probably never be 100/100, and doesn't necessarily mean your uptime score is less than 1.0
